R/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Can you add a change similar to https://github.com/Netflix/metaflow/pull/210

Reply:
Yes I have `Copyright 2020 Netflix, Inc.` in the LICENSE file.